src/main/webapp/app/tasks/default-task.service.ts

```diff
@@ -20,6 +20,10 @@ export class DefaultTaskService implements TaskService {
     return this.http.delete<void>(this.baseUrl + '/tasks/' + id);
   }
 
+  update(task: Task): Observable<void> {
```

Comment:
Also, one could argue that the `update` function should take two parameters: `update(id: string, updates: Partial<Task>)`